src/commons/collabEditing/CollabEditingHelper.ts

```diff
@@ -13,17 +13,33 @@ export function getSessionUrl(sessionId: string, ws?: boolean): string {
   return url.toString();
 }
 
+/*
 export async function checkSessionIdExists(sessionId: string): Promise<boolean> {
   const resp = await fetch(getSessionUrl(sessionId));
 
   return resp && resp.ok;
 }
+*/
```

Comment:
It seems like we no longer need this function. Can we delete it?

Reply:
Yes.